src/lua/oltp_common.lua

```diff
@@ -117,6 +122,12 @@ function cmd_load()
    end
 end
 
+function execute_multi_value_insert()
+   local drv = sysbench.sql.driver()
```

Comment:
Why are you creating a connection every time execute_multi_value_insert is called. Connection should be created only once. It is already being created in the thread_init so you might have con available here already.